recipes/dtaidistance/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "dtaidistance" %}
+{% set version = "2.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9d9b5be57f1abdac504e271be2d1c9d10cdd7b5cf30b435dfba8b05090a531b6
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - cython
+    - numpy
+  host:
+    - pip
+    - python
+    - setuptools
+    - wheel
+  run:
+    - {{ pin_compatible('numpy') }}
+    - cython
+    - matplotlib
```

Since it doesn't seem like a qt-based GUI is needed, better to take the bot up on its suggestion and depend on matplotlib-base:
```suggestion
    - matplotlib-base
```